src/main.rs

```diff
@@ -46,6 +46,13 @@ fn parse_matches() -> ArgMatches<'static> {
                 .long("debug-build")
                 .help("Enable debug build"),
         )
+        .arg(
+            Arg::with_name("release_channel")
+                .long("release-channel")
+                .takes_value(true)
+                .required_if("deploy_method", "tar") // Require if deploy_method is "tar"
+                .help("release version. e.g. v1.17.2. Required if '--deploy-method tar'"),
+        )
```

Comment:
From my comment on the last review, it might make more sense to change it so that `--local-path` and `--release-channel` conflict with each other, then you don't need `--deploy-method`

And in fact, the `DeployMethod` enum could look something like:

```
enum DeployMethod {
    Local(String),
    ReleaseChannel(String),
    Skip,
}
```